errors package testing strictly expects the cloned repo folder name to be `defradb`

Context: Time to time I do some experiments / tests on a separate clone (with our default remote) of `defradb`, that I locally call `clefradb`. Sometimes just want to purge everything and want a clean state clone to test things.

Seems like the `tests` package expects the following patterns to match:
`"^gndjdhs\. Stack: .*\/defradb\/errors\/errors_test\.go:[0-9]+ \([a-zA-Z0-9]*\)"`
`"^gndjdhs\. Kv1: 1, Kv2: 2\. Stack: .*\/defradb\/errors\/errors_test\.go:[0-9]+ \([a-zA-Z0-9]*\)"`

Which means a simple `git clone https://github.com/sourcenetwork/defradb clefradb` and running the test will fail with:

```
=== Failed
=== FAIL: errors TestErrorFmtvWithStacktrace (0.01s)
    errors_test.go:148:
                Error Trace:    /home/shahzad/~~/clefradb/errors/errors_test.go:148
                Error:          Expect "gndjdhs. Stack: /home/shahzad/~~/clefradb/errors/errors_test.go:141 (0x676770)
                                        TestErrorFmtvWithStacktrace: err := New(errorMessage)
                                /home/shahzad/.local/bin/go1.18.5/src/testing/testing.go:1439 (0x53e414)
                                        tRunner: fn(t)
                                /home/shahzad/.local/bin/go1.18.5/src/runtime/asm_amd64.s:1571 (0x49c3e1)
                                        goexit: BYTE    $0x90   // NOP
                                " to match "^gndjdhs\. Stack: .*\/defradb\/errors\/errors_test\.go:[0-9]+ \([a-zA-Z0-9]*\)"
                Test:           TestErrorFmtvWithStacktrace

=== FAIL: errors TestErrorWithStack (0.01s)
    errors_test.go:48:
                Error Trace:    /home/shahzad/~~/clefradb/errors/errors_test.go:48
                Error:          Expect "gndjdhs. Stack: /home/shahzad/~~/clefradb/errors/errors_test.go:41 (0x675938)
                                        TestErrorWithStack: errWithStack := WithStack(err)
                                /home/shahzad/.local/bin/go1.18.5/src/testing/testing.go:1439 (0x53e414)
                                        tRunner: fn(t)
                                /home/shahzad/.local/bin/go1.18.5/src/runtime/asm_amd64.s:1571 (0x49c3e1)
                                        goexit: BYTE    $0x90   // NOP
                                " to match "^gndjdhs\. Stack: .*\/defradb\/errors\/errors_test\.go:[0-9]+ \([a-zA-Z0-9]*\)"
                Test:           TestErrorWithStack

=== FAIL: errors TestErrorFmtvWithStacktraceAndKvps (0.03s)
    errors_test.go:166:
                Error Trace:    /home/shahzad/~~/clefradb/errors/errors_test.go:166
                Error:          Expect "gndjdhs. Kv1: 1, Kv2: 2. Stack: /home/shahzad/~~/clefradb/errors/errors_test.go:160 (0x676a77)
                                        TestErrorFmtvWithStacktraceAndKvps: err := New(errorMessage, NewKV("Kv1", 1), NewKV("Kv2", "2"))
                                /home/shahzad/.local/bin/go1.18.5/src/testing/testing.go:1439 (0x53e414)
                                        tRunner: fn(t)
                                /home/shahzad/.local/bin/go1.18.5/src/runtime/asm_amd64.s:1571 (0x49c3e1)
                                        goexit: BYTE    $0x90   // NOP
                                " to match "^gndjdhs\. Kv1: 1, Kv2: 2\. Stack: .*\/defradb\/errors\/errors_test\.go:[0-9]+ \([a-zA-Z0-9]*\)"
                Test:           TestErrorFmtvWithStacktraceAndKvps

DONE 924 tests, 3 failures in 24.562s
make: *** [Makefile:146: test] Error 1
```

Should be an easy fix to make the directory name be more flexible (i.e. detect the current cloned directory name).

--- a/errors/errors_test.go
+++ b/errors/errors_test.go
@@ -45,7 +45,7 @@ func TestErrorWithStack(t *testing.T) {
 	// Assert that the first line starts with the error message and contains this [test] function's stacktrace-line -
 	// including file, line number, and function reference. An exact string match should not be used as the stacktrace
 	// is machine dependent.
-	assert.Regexp(t, fmt.Sprintf("^%s\\. Stack: .*\\/defradb\\/errors\\/errors_test\\.go:[0-9]+ \\([a-zA-Z0-9]*\\)", errorMessage), result)
+	assert.Regexp(t, fmt.Sprintf("^%s\\. Stack: .*\\/.*\\/errors\\/errors_test\\.go:[0-9]+ \\([a-zA-Z0-9]*\\)", errorMessage), result)
 
 	// Assert that the error contains this function's name, and a print-out of the generating line.
 	assert.Regexp(t, "TestErrorWithStack: errWithStack := WithStack\\(err\\)", result)
@@ -145,7 +145,7 @@ func TestErrorFmtvWithStacktrace(t *testing.T) {
 	// Assert that the first line starts with the error message and contains this [test] function's stacktrace-line -
 	// including file, line number, and function reference. An exact string match should not be used as the stacktrace
 	// is machine dependent.
-	assert.Regexp(t, fmt.Sprintf("^%s\\. Stack: .*\\/defradb\\/errors\\/errors_test\\.go:[0-9]+ \\([a-zA-Z0-9]*\\)", errorMessage), result)
+	assert.Regexp(t, fmt.Sprintf("^%s\\. Stack: .*\\/.*\\/errors\\/errors_test\\.go:[0-9]+ \\([a-zA-Z0-9]*\\)", errorMessage), result)
 
 	// Assert that the error contains this function's name, and a print-out of the generating line.
 	assert.Regexp(t, "TestErrorFmtvWithStacktrace: err := New\\(errorMessage\\)", result)
@@ -163,7 +163,7 @@ func TestErrorFmtvWithStacktraceAndKvps(t *testing.T) {
 	// Assert that the first line starts with the error message and contains this [test] function's stacktrace-line -
 	// including file, line number, and function reference. An exact string match should not be used as the stacktrace
 	// is machine dependent.
-	assert.Regexp(t, fmt.Sprintf("^%s\\. Kv1: 1, Kv2: 2\\. Stack: .*\\/defradb\\/errors\\/errors_test\\.go:[0-9]+ \\([a-zA-Z0-9]*\\)", errorMessage), result)
+	assert.Regexp(t, fmt.Sprintf("^%s\\. Kv1: 1, Kv2: 2\\. Stack: .*\\/.*\\/errors\\/errors_test\\.go:[0-9]+ \\([a-zA-Z0-9]*\\)", errorMessage), result)
 
 	// Assert that the error contains this function's name, and a print-out of the generating line.
 	assert.Regexp(t, "TestErrorFmtvWithStacktraceAndKvps: err := New\\(errorMessage, NewKV\\(\"Kv1\", 1\\), NewKV\\(\"Kv2\", \"2\"\\)\\)", result)

